engine/schema/src/com/cloud/vm/dao/DomainRouterDaoImpl.java

```diff
@@ -432,4 +432,12 @@ public void removeRouterFromGuestNetwork(final long routerId, final long guestNe
         sc.setParameters("states", State.Stopped);
         return listBy(sc);
     }
+
+    @Override
+    public List<DomainRouterVO> listIncludingRemovedByVpcId(long vpcId) {
+        SearchCriteria<DomainRouterVO> sc = VpcSearch.create();
```

Comment:
These are not column names, these are the names that are given inside that VpcSearch object. This is how it's done in all DAOs :)
```java
        VpcSearch = createSearchBuilder();
        VpcSearch.and("role", VpcSearch.entity().getRole(), Op.EQ);
        VpcSearch.and("vpcId", VpcSearch.entity().getVpcId(), Op.EQ);
        VpcSearch.done();
```